<launch>
    <!-- Define parameters -->
    <param name="camera_fov_deg" value="95" />
    <param name="camera_resolution" value="256" />
    <param name="detection_method" value="standalone_euclidean_clustering" />
    <param name="visualize" value="true" />
    <param name="clustering_distance_threshold" value="0.15" />
    <param name="clustering_min_points" value="2" />
    <param name="clustering_max_points" value="30" />
    <param name="camera_poses_topic" value="camera_poses" />

    <!-- Specify the node to be launched -->
    <node pkg="mecators_sensor_fusion_ros_pkg" type="LidarDetectorNode" name="lidar_detector">
        <!-- Remap topics if necessary -->
        <remap from="oak" to="oak" />
        <remap from="scan" to="scan" />
        <remap from="lidar_poses" to="lidar_poses" />
        <remap from="~detected_robots" to="~detected_robots" />
    </node>
</launch>
